src/main/webapp/about.jsp

```diff
@@ -64,6 +64,14 @@
         This page should also be used to describe the features and improvements
         you've added.
       </p>
+
+      <h1>About Team int ellects;</h1>
+      <h3>Our Members: Ivy, Jose, Marielle, and Silvie</h3>
+      <p>
```

Comment:
I think you probably want a closing `</p>` tag at the end of the text.

Or you might consider making each section its own paragraph, by wrapping each line in `<p> </p>`.